settings.gradle

```diff
@@ -1,7 +1,6 @@
 pluginManagement {
   repositories {
     mavenLocal()
-    maven { url "https://dl.bintray.com/spinnaker/gradle/" }
```

Comment:
bintray is not supported anymore
<img width="1144" alt="Screen Shot 2022-06-02 at 11 19 26 AM" src="https://user-images.githubusercontent.com/43676929/173156408-e050813b-95ae-48ce-9743-fe7aa47c2c59.png">